SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularitySmtpSender.java

```diff
@@ -62,6 +68,10 @@ public void stop() {
   }
 
   public void queueMail(final List<String> toList, final List<String> ccList, final String subject, final String body) {
+    if (LOG_EMAILS_FOR_DEBUG) {
+      LOG.trace("TheMail: " + body);
```

Comment:
We can do without the `LOG_EMAILS_FOR_DEBUG`. We can control the logging level for each class in the configuration for Singularity, so just putting it at trace should be enough.

Also for logging, you can so something more like this rather that using "string" + "string"
```java
LOG.trace("Sending email: {}, to: {}, cc: {}", body, toList, ccList)
```